msteams-platform/apps-in-teams-meetings/teams-live-share-faq.md

```diff
@@ -59,7 +67,57 @@ No. Teams currently doesn't support encrypted media for tab applications.
+
+<br>
+
+</details>
+
+<details>
+<summary><b>Can I use multiple Fluid containers?</b></summary>
+
+Currently, Live Share only supports having one container using our provided Azure Fluid Relay service. However, it is possible to use both a Live Share container and a container created by your own Azure Fluid Relay instance.
+
+<br>
+
+</details>
+
+<details>
+<summary><b>Can I change my Fluid container schema after creating the container?</b></summary>
+
+Currently, Live Share does not support adding new `initialObjects` to the Fluid `ContainerSchema` after creating or joining a container. Because Live Share sessions are short-lived, this is most commonly an issue during development after adding new features to your app.
+
+> [!NOTE]
+> If you are using the `dynamicObjectTypes` property in the `ContainerSchema`, you can add new types at any point. If you later remove types from the schema, existing DDS instances of those types will gracefully fail.
+
+To fix errors resulting from changes to `initialObjects` when testing locally in your browser, remove the hashed container ID from your URL and reload the page. If you are testing in a Teams meeting, start a new meeting and try again.
+
+If you plan to update your app with new `SharedObject` or `EphemeralObject` instances frequently, you should consider how you deploy new schema changes to production. While the actual risk is relatively low and short lasting, there may be active sessions at the time you roll out the change. Existing users in the session should not be impacted, but users joining that session after you deployed a breaking change may have issues connecting to the session. To mitigate this, you may consider some of the following solutions:
+
+- Deploy schema changes for your web application outside of normal business hours.
+- Use `dynamicObjectTypes` for any changes made to your schema, rather than changing `initialObjects`.
+- Versioning your schema for each session. *Note:* Live Share API to do this, and difficulty may vary for your project.
+
+<br>
+
+</details>
+
+<details>
+<summary><b>Are there limits to how many change events I can emit through Live Share?</b></summary>
+
+While Live Share is in Preview, we do not enforce any limit to events emitted through Live Share. For optimal performance, you should debounce changes emitted through `SharedObject` or `EphemeralObject` instances to one message per 50ms or more. This is especially important when sending changes based on mouse or touch coordinates, such as when synchronizing cursor positions, inking, and dragging objects around a page.
+
+> [!IMPORTANT]
+> In the future, formal rate limiting may be added to the Live Share service. We are monitoring this situation continually. Any update to these terms will be added here in advance.
```

Comment:
Recommend removing this information "In the future, formal rate limiting may be added to the Live Share service. We are monitoring this situation continually. Any update to these terms will be added here in advance." 

as we don't state something that would be available in future.